project/firstproject/src/main/resources/static/javascript/scripts.js

```diff
@@ -9,11 +9,29 @@ function loginUser()
         url: "/auth/login",
         //data:{email: email, password: password},
         data: JSON.stringify(userDTO),
-        success: function (){
+        success: function () {
             window.location.replace("http://localhost:8080/auth/logged-in");
         },
-        error: function (e){
+        error: function (e) {
             document.getElementById("errorMessage").innerHTML = e.responseText;
         }
     });
+}
+
+function saveQuestion() {
```

Comment:
Minor: Split in different .js files the ajax call for login vs the ajax call for question.